include/kseq_load.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * @file kseq_load.hpp
+ * @brief Function and variable definitions for kseq_load.cpp
+ * @parblock
+ * SortMeRNA - next-generation reads filter for metatranscriptomic or total RNA
+ * @copyright 2012-16 Bonsai Bioinformatics Research Group
+ * @copyright 2014-16 Knight Lab, Department of Pediatrics, UCSD, La Jolla
+ *
+ * This file is part of SortMeRNA.
+ *
+ * SortMeRNA is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * SortMeRNA is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with SortMeRNA. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @authors jenya.kopylov@gmail.com
+ *          laurent.noe@lifl.fr
+ *          helene.touzet@lifl.fr
+ *          pierre.pericard@lifl.fr
+ *          mikael.salson@lifl.fr
+ *          robknight@ucsd.edu
+ *
+ */
+ 
+#ifndef KSEQ_LOAD_H
+#define KSEQ_LOAD_H
+
+#include "common.hpp"
+#include "kseq.h"
+#ifdef HAVE_LIBZ
+#include <zlib.h>
+KSEQ_INIT(gzFile, gzread)
+#else
+KSEQ_INIT(int, read)
+#endif
+
+
+
+/*! @fn load_reads()
+    @brief load reads into buffer using kseq library
```

Comment:
I just modified the input from ```char* raw``` to ```char*& raw``` since I'm freeing the memory outside the function. Also, in case we need to modify the pointer later on. Thanks for `pointing` my attention to this :)